lib/fluent/plugin_helper/inject.rb

```diff
@@ -0,0 +1,112 @@
+#
+# Fluentd
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+require 'fluent/event'
+require 'time'
+
+module Fluent
+  module PluginHelper
+    module Inject
+      def inject_record(tag, time, record)
```

Comment:
`inject_record` is misleading name.
`inject` is used like `inject xxx into yyy` so this name seems to mean `inject record into yyy`.
So we should use other name, `inject_value_to_record` or something.